next.config.js

```diff
@@ -7,6 +7,11 @@ const nextConfig = {
         hostname: '127.0.0.1',
         port: '54321',
         pathname: '/storage/v1/object/public/**'
+      },
+      {
+        protocol: 'https',
```

Comment:
This setting is needed to display uploaded images on production environment.

If the setting is nothing, Next.js automatically add link to image_url like `https://xxx.com/_next/image?referer=https://image_object_url` .